### PR TITLE
Integrate django-cms into our project

### PR DESCRIPTION
**Pull request by Baschdl**
Part 1/n for #25: This PR only includes the setup, we have to integrate our sites separately into django-cms. Open question: how to automatically load the initial pages?

The user interface is available at [http://localhost:8000/admin](http://localhost:8000/admin) and you can login with a superuser

### CODE REVIEW

File: backend/requirements.txt [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
+djangocms-style==2.3.0
+djangocms-video==2.3.0
+djangocms_text_ckeditor==3.9.1
 djangorestframework==3.11.0
 djangorestframework-filters==0.11.1
 JSON-log-formatter==0.3.0
 numpy==1.18.2
 pandas==1.0.3
 pre-commit==2.2.0
+psycopg2==2.8.5
```
Comment (feeds) [inline backend/requirements.txt]: Do we really need this?

**Review by feeds — CHANGES_REQUESTED** [review]
Otherwise the setup works for me. Should there anything else that should be achievable with this state other than the app not crashing?

**Review by feeds — APPROVED** [review]

File: backend/requirements.txt (hunk 1 above)
Comment (Baschdl) [reply backend/requirements.txt]: Moved to production requirements

File: backend/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:18.04
 
-RUN apt-get update && apt-get install -y python3 python3-pip libpq-dev gettext
+RUN apt-get update && apt-get install -y python3 python3-pip libpq-dev gettext python-dev
```
Comment (maltezacharias) [inline backend/Dockerfile:3]: What is/was this change needed for?

Comment (Baschdl) [reply backend/Dockerfile:3]: I think one of the django-cms plugins needed it but I didn't try it without it. No idea why they would need python2.
